<HTML><HEAD><TITLE> RESULTS</TITLE> </HEAD><BODY BGCOLOR="FFFFF0"><b></b> 
<h2>
<a href="http://www.biomedcentral.com/1471-2105/10/345">
BayesPI -- a new model to study protein-DNA interactions: a case study of condition-specific protein binding parameters for Yeast transcription factors
<p></A></b> </h2> 
<br> 

<b><A HREF="mailto:junbai.wang@rr-research.no">Junbai Wang</a>* </b>
and Morigen** <br><br> 
<b>
*Division of Pathology<br>
**Department of Cell Biology, Institute for Cancer Research
<br>
The Norwegian Radium Hospital, Rikshospitalet University Hospital
<br>
Montebello 0310 Oslo, Norway

</b> <p> <br>
We have incorporated Bayesian model regularization with biophysical modeling of protein-DNA interactions (i.e. a Femi-Dirac form of protein binding probability) and of genome-wide nucleosome positioning to study protein-DNA interaction using high-throughput dataset. The newly-developed method (BayesPI) includes the estimation of transcription factor (TF) binding energy matrices, the computation of binding affinity of a TF target site and the corresponding chemical potential. The method was successfully tested on synthetic ChIP-chip datasets, real yeast ChIP-chip experiments and human ChIP-Seq data. Subsequently, it was used to estimate condition-specific and species-specific protein-DNA interaction for several yeast TFs. The results showed that the adaptive modification of protein binding parameters and the variation of individual nucleotide affinity in either recognition or flanking sequences may play pivotal roles in environment-specific binding patterns of yeast TFs and in divergence of TF binding sites across three related yeast species (i.e. S. cerevisiae, S. mikatae, and S. bayanus).
</br>

